pkg/wfc3/calwf3/calwf3/wf3member.c

```diff
@@ -43,10 +42,9 @@ WF3Info *wf3		o: exposure specific flags and info
 
 	/* Initialize local strings */
 	rootname[0] = '\0';
-	outroot[0]  = '\0';
 	mtype[0]    = '\0';
 
-	strcpy (outroot, asn->product[prodid].subprod[posid].name);
+	const char * outroot = asn->product[prodid].subprod[posid].name;
```

Comment:
The idea behind a code review is that the reviewer also thinks about it - otherwise what is the point? This change is also in calacs, in identical code, if you have a concern here you should have an identical concern there - do you?